src/rospkg/os_detect.py

```diff
@@ -693,6 +711,7 @@ def get_codename(self):
 OS_LINARO = 'linaro'
 OS_MINT = 'mint'
 OS_NEON = 'neon'
+OS_OPENEMBEDDED = 'oe'
```

Comment:
As a side comment, `oe` is a little short and hard to understand if you don't know what you're looking at, but `openembedded` is really long. So I don't have a better suggestion.

Reply:
Agree; will make it openembedded so that's explicit for everybody.